document/CoreFormat.mdk

```diff
@@ -356,65 +356,60 @@ source to allow the run to be repeated.
 A `run` object should contain a property named `files` whose value is a JSON object,
 each of whose properties represents a file that was scanned in the course of the run.
 
-Each property name in the `files` object shall be the URI of a scanned file.
-No two of these property names shall be equivalent as defined in &sect;[#uriProperties].
-
-Each property value in the `files` object shall be an array of `file` objects (&sect;[#file])
-which contain information about the file identified by the URI in the property name.
-
 The object specified by the `files` property should contain properties representing at least those files in which results were detected,
 but it may contain properties representing all scanned files (whether or not results were detected in those files),
```

Comment:
'files' can represent absolutely anything pointed to by a 'uri' in the format. Whether by a notification, a resultFile, etc.